sapbtp-operator-charts/templates/crd.yml

```diff
@@ -765,6 +768,9 @@ spec:
               ready:
                 description: Indicates whether instance is ready for usage
                 type: string
+              shared:
+                description: Indicates whether instance is shared
```

Comment:
ua review

Reply:
Whether the instance is shared. Default value is false.